cmd/detectExecuteScan.go

```diff
@@ -499,7 +499,7 @@ func isMajorVulnerability(v bd.Vulnerability) bool {
 
 func postScanChecksAndReporting(ctx context.Context, config detectExecuteScanOptions, influx *detectExecuteScanInflux, utils detectUtils, sys *blackduckSystem) error {
 	errorsOccured := []string{}
-	vulns, components, err := getVulnsAndComponents(config, influx, sys)
+	vulns, _, err := getVulnsAndComponents(config, influx, sys)
```

Comment:
can you please change the method getVulnsAndComponents ? Remove the return params.

Reply:
Done